src/Psa/Output.purs

```diff
@@ -155,15 +155,15 @@ shouldShowError { filterCodes, censorCodes, censorSrc, censorLib, censorWarnings
 
 errorPath :: Array String -> String -> String -> PsaPath
 errorPath libDirs path short =
-  if any (path `startsWith` _) libDirs
+  if any (\dir -> path `startsWith` Pattern dir) libDirs
```

Comment:
Since we've already imported `Str`, can we just use `Str.Pattern` like you use in other modules?